code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -146,7 +151,9 @@
 
 /obj/structure/closet/crate/secure/New()
 	..()

```

Comment:
I should've mentioned I meant this entire statement can go; all it does is call parent.